src/application/sidebar/ExplorerPanel.tsx

```diff
@@ -46,39 +49,39 @@ export function ExplorerPanel() {
         title="Open Files"
       >
         {hasLeftAndRightPanelsFiles && <div className="ml-4 text-xs font-bold">LEFT</div>}
-        {left.openEditors.length > 0 && (
+        {leftPaneOpenEditors.length > 0 && (
```

Comment:
I don't think it would be useful here. The hook exists for components that only need the number of open editors, like `MainPanel`. Here the component needs to have access to the list of open editors anyway, so it might as well use the length. Maybe `ExplorerPanel` should just check the editors count and render ExplorerList with a paneId prop. Then the `ExplorerList`s would call `useOpenEditorsDataForPane`